### DISCUSSION TITLE
Welcome to Any Aspect Ratio Discussions!

### DISCUSSION THREAD

**Discussion by fayaz12g** [fayaz12g/totk-aar#30] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to hear your thoughts and suggestions!
  * Have a feature request? Put it here!
  * Have a suggestion for the code? Talk about it here (or put in a pull request)
  * Feel free to talk about anything to do with this tool, like what you don't like about it!

**Comment by gaklimited** [comment]
The HUD isn't stretched at 16:10, which is excellent, but I wanted to know if the space between the HUD and the top/bottom of the screen can be adjusted.

<img width="510" alt="2023-06-21 (03h31m47s) yuzu  yuzu_1474__The_Legend_of_Zelda_Tears_of_the_Kingdo" src="https://github.com/fayaz12g/totk-aar/assets/66001816/bb5aa991-4439-4c48-9357-fbfbf840b264">
16:9 (without AAR)
<img width="525" alt="2023-06-21 (03h22m52s) yuzu  yuzu_1474__The_Legend_of_Zelda_Tears_of_the_Kingdo" src="https://github.com/fayaz12g/totk-aar/assets/66001816/c24f16e0-48c5-49ff-8fba-bc1dbd766635">
16:10 (with AAR)

**Reply by fayaz12g** [reply]
Not sure if you've been keeping up with the changes, but the latest version fixes this :)

**Reply by gaklimited** [reply]
Yeah, I was. I had reacted on the 8.0.0 release. Thanks for implementing it :)

I even switched from using this in conjunction with MaxLastBreasth's Totk Mod Folder over to this entirely since it covers Blackscreenfix, a more updated dfps and visual improvements too.

Seriously great work man!

**Comment by cprogpb** [comment]
I haven't touched python, but I'd like to try it, please ask what program your source code is packaged with

**Reply by cprogpb** [reply]
I would like to translate the GUI and post it on our country's community website, of course I will indicate the source

**Reply by fayaz12g** [reply]
Hey there! I use pyinstaller to package
